python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2896,15 +2903,29 @@ def _parse_import_prerequisites(self, graph):
         """
         missing_operators = set()
         for node in graph.node:
+            try:
+                from tensorflow.python.framework import op_def_registry
+            except ImportError as e:
+                raise ImportError(
+                    "Unable to import tensorflow which is required {}".format(e))
+            getOpDef = op_def_registry._registered_ops.get if hasattr(op_def_registry,\
+                        "_registered_ops") else op_def_registry.get
+            op_def = getOpDef(node.op)
             if node.op == "Placeholder" or node.op == 'PlaceholderWithDefault':
                 pass
             elif node.op == "Const":
                 pass
+            elif node.op in ["PartitionedCall", "StatefulPartitionedCall"]:
```

Comment:
Should note TF version here to before supporting.

Reply:
@srkreddy1238, could you please elaborate more here? Do you want us to check if these ops are supported in different TF versions?

Reply:
@srkreddy1238 ,
Also we have tested this on two machine one with TF 1.X & other with TF 2.X, and this condition works fine on both. So, I am assuming no work to be done here in that case.